sanic/server.py

```diff
@@ -139,8 +139,10 @@ def connection_timeout(self):
                 self._request_stream_task.cancel()
             if self._request_handler_task:
                 self._request_handler_task.cancel()
-            exception = RequestTimeout('Request Timeout')
-            self.write_error(exception)
+            try:
```

Comment:
How is this try/except logic different from the current situation? To me it appears identical, since any other exception would get raised with this implementation, and you do the same thing in the except clause.